### ISSUE TITLE
Consider registering /contribute.json with .well-known

### ISSUE THREAD

**Issue by pmclanahan** [mozilla/contribute.json#30]
http://tools.ietf.org/html/rfc5785

**Comment by gene1wood** [comment]
Alternatively, you could begin using a location like /.well-known/contribute.json while you wait to register the name.

**Comment by marumari** [comment]
I would really like this; it honestly probably should have been /.well-known/contribute.json in the first place, but the cat is a bit out of the bag on this one.  :)

Once it's registered, I'd be happy to have the Observatory check for both.

**Comment by pmac** [comment]
I've sent an email to the registration list-serv per instructions in the linked spec. I'll update the issue with results.

**Comment by marumari** [comment]
Any updates on this at all, @pmac?

**Comment by pmac** [comment]
Yes. As you can see from the [mailing-list archive](https://mailarchive.ietf.org/arch/search/?email_list=wellknown-uri-review) I think we've been rejected. I tried explaining further, but there's been no reply since the 10th. We can use /.well-known/contribute.json seemingly with the knowledge that such a thing should not be officially registered and thus not conflict with our use, but it seems we'll have to do so unofficially if we use it.

**Comment by marumari** [comment]
Yeah, I think that's fine.  I can think of two good ways of doing it:

1. `/.well-known/contribute.json`
2. `<meta name="contribute" content="/foo/bar/contribute.json">`

**Comment by pmac** [comment]
I like \#1 myself. I think forcing clients that want to discover these files to parse HTML is a bit much. I could be convinced otherwise though if there's a compelling advantage. I think I still like the `/.well-known/contribute.json` approach that then optionally points to other files as necessary.

**Comment by strugee** [comment]
One problem with `.well-known` is that it makes certain things impossible. E.g. if you serve two webapps at different paths on one domain, who gets `.well-known`? Just to point out the other side of this.

That being said, I still think 1. might be the best one. It's easier to implement for clients.

**Comment by jeffbryner** [comment]
Is it worth bringing up the time period in: https://tools.ietf.org/html/rfc5785#section-5.1 ?

**Comment by jeffbryner** [comment]
Is it worth bringing up the time period in: https://tools.ietf.org/html/rfc5785#section-5.1 ?
